src/components/sequencing/SequenceEditor.svelte

```diff
@@ -265,10 +265,10 @@
     const sequence = viewUpdate.state.doc.toString();
     disableCopyAndExport = sequence === '';
     const tree = syntaxTree(viewUpdate.state);
-    const output = await selectedOutputFormat?.toOutputFormat?.(tree, sequence, commandDictionary, sequenceName);
+    let output = await selectedOutputFormat?.toOutputFormat?.(tree, sequence, commandDictionary, sequenceName);
 
     if ($sequenceAdaptation?.modifyOutput !== undefined && output !== undefined) {
-      $sequenceAdaptation?.modifyOutput(output, parameterDictionaries, channelDictionary);
+      output = $sequenceAdaptation?.modifyOutput(output, parameterDictionaries, channelDictionary);
```

Comment:
The type return for `modifyOutput` for the `ISequenceAdaptation` interface is `any`, while the `toOutputFormat` return is a string. Can we tighten down the type of `modifyOutput` to return a `string` instead of `any`? What are the implications in doing so?

Reply:
No real implications as the output needs to be a string as this is what is displayed in the bottom editor.

Reply:
All that I want to ensure is that the components that depend on `output` are able to properly handle whatever gets spit out from `modifyOutput`. If `output` is only meant to be a string, then we should maybe have something that converts the output of `modifyOutput` to a string before setting it to `output`

Reply:
Matt pointed out that it's reasonable to rename this since it no longer modifies, perhaps renaming to transform would be more clear